dev/devicelab/test/run_test.dart

```diff
@@ -12,45 +13,54 @@ import 'common.dart';
 
 void main() {
   const ProcessManager processManager = LocalProcessManager();
+  final String dart = path.absolute(
+    path.join('..', '..', 'bin', 'cache', 'dart-sdk', 'bin', 'dart'));
 
   group('run.dart script', () {
-    Future<ProcessResult> runScript(List<String> testNames,
+    // The tasks here refer to files in ../bin/tasks/*.dart
+
+    Future<ProcessResult> runScript(List<String> taskNames,
         [List<String> otherArgs = const <String>[]]) async {
```

Comment:
nit (unrelated to this change): formatting here is kinda strange.

Reply:
What would you prefer?